pkg/converter/converter.go

```diff
@@ -65,6 +65,33 @@ func composeServiceVolumesToK8s(
 	return volumes, volumeMounts
 }
 
+func composeServiceTmpfsToK8s(
+	container string,
+	paths []string,
+) (map[string]core.Volume, []core.VolumeMount) {
+	volumeMounts := []core.VolumeMount{}
+	volumes := make(map[string]core.Volume)
+
+	for _, path := range paths {
+		name := fmt.Sprintf("%s-tmpfs-%s", container, strings.ReplaceAll(path, "/", "-"))
+		name = strings.ReplaceAll(name, "--", "-")
```

Comment:
I don't think that's sufficient to collapse multiple strings. Think `//foo////bar`